packages/grafana-ui/src/components/Logs/LogMessageAnsi.tsx

```diff
@@ -82,6 +82,7 @@ export class UnThemedLogMessageAnsi extends PureComponent<Props, State> {
     return chunks.map((chunk, index) => {
       const chunkText = this.props.highlight?.searchWords ? (
         <Highlighter
+          key={index}
```

Comment:
This is unrelated change, but I noticed it when working on this.  Unfortunately, there is no unique identifier, so I ended up with index.